blue_only.py

```diff
@@ -123,6 +137,10 @@ def main() -> int:
     loop_counter = 0
     last_time = current_milli_time()
     last_image_time = current_milli_time()
+
+    if args.method == "yolo":
+        yolo_model = yolo_decision.DetectLandingPad("cpu", 0.5, "models/best-2n.pt")

```

Comment:
let's add these parameters to a config file (look at config.yaml in computer-vision-python for an example)